modules/oe_theme_demo/config/optional/block.block.oe_theme_demo_header_language_switcher.yml

```diff
@@ -0,0 +1,19 @@
+langcode: en
```

Comment:
@richardcanoe the demo language switcher block needs to be removed as it is now provided by OE Multilingual. See #70